papers/oliver_beckstein/oliver_beckstein.rst

```diff
@@ -90,9 +90,28 @@ The MDAnalysis library :cite:`Michaud-Agrawal:2011fu` addresses this problem by
 
-MDAnalysis is available in source form under the GNU General Public License v2 from GitHub https://github.com/MDAnalysis/mdanalysis and PyPi;  conda packages are also available. The documentation is extensive http://docs.mdanalysis.org including an introductory tutorial http://www.mdanalysis.org/MDAnalysisTutorial/ and a very friendly and welcoming user and developer community. 
+MDAnalysis is available in source form under the GNU General Public License v2 from GitHub https://github.com/MDAnalysis/mdanalysis and PyPi;  conda packages are also available. The documentation is extensive http://docs.mdanalysis.org including an introductory tutorial http://www.mdanalysis.org/MDAnalysisTutorial/ and a very friendly and welcoming user and developer community.
 
+Analysis Module
+---------------
 
+In the MDAnalysis.analysis module we provide a large variety of standard analysis algorithms, like RMSD, alignment, native contacts, as well as unique algorithms, like the LeaftleftFinder and Path Similarty Analysis. We have recently started to unify the interface to the different algorithms with an `AnalysisBase` class. Currently PersistenceLength, InterRDF, LinearDensity and Contacts analysis have been ported. If applicable we also strive to make the API's to the algorithms generic. Most other tools hand the user analysis algorithms as black boxes, we want to avoid that and give the users all he needs to adapt an analysis to his/her needs.
+
+The new Contacts class is a good example a generic API that allows easy adaptations of algorithms while still offering an easy setup for standard analysis types. For contact analysis it is possible to have different metrics based on coordinates alone :cite:`Best2013,Franklin2007`. We have designed the API to choose between common metrics and pass user defined functions to develop new metrics. This generic interface allowed us to implement a q1q2 analysis ontop of the Contacts class. Below is incomplete code example that shows how to implement a q1q2 analysis, a more detailed explanatain can be found in the docs. **how to generate a link/should I generate a link**
+
```

Comment:
... a section with a link to the docs already exists.

Generally speaking, I find that papers, as the fairly permanent record, are the place to explain algorithms and "unchanging" ideas so I rather put it in a paper to start with and then reference the paper.